### ISSUE TITLE
Version 2.0+ (that starts using ImageProcessing gem) increases image sharpness and increases file size

### ISSUE THREAD

**Issue by feliperaul** [carrierwaveuploader/carrierwave#2481]
I know version 2.0.0 is almost one year old by now, but I only managed to migrate from 1.3.1 this weekend.

I expected no change in image quality and file size when upgrading from 1.3.1, but what I saw was a visible image degradation (due to excessive artificial sharpening) and increased file size after the migration.

I'm going to show 2 pictures as examples. I recommend opening Carrierwave's versions in new tabs so you can see the images in full size to notice the difference.

Original file (66kb):

![image](https://user-images.githubusercontent.com/743879/82135488-adee6280-97d9-11ea-90bb-4ada71eb1e64.png)

Uploaded with Carrierwave 1.3.1 (66kb):

![image](https://user-images.githubusercontent.com/743879/82135495-be064200-97d9-11ea-8454-85572ca91aab.png)

Uploaded with Carrierwave 2.0+ (72kb):

![image](https://user-images.githubusercontent.com/743879/82135508-dece9780-97d9-11ea-9b49-21639ee87d8e.png)

Now picture #2

Original file (386kb):

![image](https://user-images.githubusercontent.com/743879/82135509-e7bf6900-97d9-11ea-8674-e2cecf5bc641.png)

Uploaded with Carrierwave 1.3.1 (385kb):
![image](https://user-images.githubusercontent.com/743879/82135515-f7d74880-97d9-11ea-8486-0931312b8df8.png)

Uploaded with Carrierwave 2.0+ (429kb):

![image](https://user-images.githubusercontent.com/743879/82135516-04f43780-97da-11ea-970d-0cde376d36ae.png)

In both cases, there's undeniable increase of sharpness, but I didn't ask for that (even if you regard Carrierwave's 2.0+ version as better-looking than 1.3's, it's undeniable that the 1.3.1 version is **much closer to the original**, as expected, since the only processing done on the uploader was a crop).

Also, Carrierwave 2.0+ increases file sizes from 9 to 20% when compared to version 1.3.1.

I know it's a huge change in the codebase and I understand the reasoning behind it, but I think this change in quality and in sharpness was a substancial regression when comparing 1.3.1 to 2.0.0+. 

Im pinging @janko and @mshibuya since they worked on issue #2298 where this change was implemented.

**Comment by janko** [comment]
Thank you for the detailed issue report, I deeply apologize for breaking backwards compatibility. This issue has been brought up in https://github.com/janko/image_processing/pull/67, but I got side-tracked and never merged it.

This is now merged and released in ImageProcessing 1.11.0, which doesn't sharpen by default anymore.

CarrierWave could either require version `~> 1.11` from now on, or we can rely on people that are upgrading to automatically install the latest ImageProcessing gem.
